src/OSPSuite.UI/Services/DialogCreator.cs

```diff
@@ -25,9 +27,19 @@ public DialogCreator(IDialogResultToViewResultMapper mapper, DirectoryMapSetting
          _applicationConfiguration = applicationConfiguration;
       }
 
-      public void MessageBoxError(string message)
+      public void MessageBoxError(string message, bool hyperlink)
       {
-         XtraMessageBox.Show(message, _applicationConfiguration.ProductNameWithTrademark, MessageBoxButtons.OK, MessageBoxIcon.Error);
+         var args = new XtraMessageBoxArgs { Caption = _applicationConfiguration.ProductNameWithTrademark, Text = message, Buttons = new DialogResult[] {
+            DialogResult.OK } ,
+            Icon = DevExpress.Utils.Drawing.Helpers.StockIconHelper.GetWindows8AssociatedIcon(SystemIcons.Error), AllowHtmlText = DefaultBoolean.True };
```

Comment:
What is this SystemIcons.Error? What aren't you using the SuystemsIcon.Errror
Also why do you think this feature should only be available for MessageBoxError? What about Info? Or yes no etc..

Reply:
You mean use MessageBoxIcon.Error instead right? Because they are of different types. The show() method actually belongs to the base class MessageBox, not XtaMessagebox, hence the XtraMessageBoxArgs don't have it as an argument.  I used the suggested from devexpress: [](https://supportcenter.devexpress.com/ticket/details/t524421/can-i-use-messageboxicon-with-xtramessageboxargs) . 

About the rest of the MessageBoxes: of course if you also agree on actually expanding them (but then it really will have to be like this, with SystemIcon.Error and so on), I will implement it for the rest of them too, otherwise I will just write a new function for an error message box with the link, as suggested in the description of the PR.

Reply:
encapsulated this into a function